doc/for-framework-folk.rst

```diff
@@ -92,6 +92,13 @@ cause ``testtools.RunTest`` to fail the test case after the test has finished.
 This is useful when you want to cause a test to fail, but don't want to
 prevent the remainder of the test code from being executed.
 
+Exception formatting
+--------------------
+
+Testtools ``TestCase`` do their own formatting of exceptions. The attribute
```

Comment:
I think you need the word "instances" after `TestCase`, since it looks singular.

Perhaps, "`TestCase` _instances_ format their own exceptions"?